### PR TITLE
logging to syslog in appliance 

### PR DESCRIPTION
**Pull request by orenbm**
#### What does this PR do?
- logging to syslog in appliance
- logging also log tags in appliance
- renamed log_tags file

#### What ticket does this PR close?
Connected to #1059

### CODE REVIEW

**Comment by dividedmind** [comment]
(BTW, protip: if you install https://github.com/sds/overcommit it'll flag code quality violations for you as you commit.)

File: config/initializers/log_tags.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Rails.application.configure do
+
+  config.log_tags = [
+    lambda { |request| "origin=#{request.ip}" },
+    lambda { |_| "tid=#{Thread.current.object_id}" },
+    lambda { |request| "request_id=#{request.uuid}" }
```
Comment (dividedmind) [inline config/initializers/log_tags.rb]: Is there a reason for the renaming of both this field and this file?

File: config/initializers/log_tags.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  ]
+
+end
```
Comment (dividedmind) [inline config/initializers/log_tags.rb]: If you're renaming the file anyway you could add the missing trailing newline :)

File: lib/logger/formatter/conjur_formatter.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class ConjurFormatter < Logger::Formatter
-  Format = "%5s [%s] [tid=%s] [pid=%s] %s\n".freeze
+  Format = "%5s %s [tid=%s] [pid=%s] %s\n"
```
[the remark this answers is not in the document]
Comment (orenbm) [reply lib/logger/formatter/conjur_formatter.rb]: this is the oss formatter. i just removed the brackets from the severity

File: config/initializers/log_tags.rb (hunk 1 above)
Comment (orenbm) [reply config/initializers/log_tags.rb]: what do you mean by renaming the field?

I renamed the file from `logging` to `log_tags` as it is more specific to what's going on in this file.

**Pull request merged**